plugins/modules/files/iso_customize.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_customize
+short_description: Add/remove/change files in ISO file
+description:
+  - This module is used to add/remove/change files in ISO file.
+  - The file inside ISO will be overwritten if it exists by option I(add_files).
+author:
+  - Yuhua Zou (@ZouYuhua) <zouy@vmware.com>
+requirements:
+  - "pycdlib"
+  - "python >= 2.7"
+version_added: '5.6.0'
```

Comment:
5.6.0 was released during the week, next one is 5.7.0
```suggestion
version_added: '5.7.0'
```

Reply:
Fix it. Thanks